scss/_pagination.scss

```diff
@@ -204,6 +204,18 @@
   // End mod
 }
 
+// Boosted mod: special style for pagination with icons only
+.icons-only {
```

Comment:
Possible solution to add correct style after the a11y fix.
Not a big fan of it since it needs a change in the markup.

Any opinion @louismaximepiton? Could it work for your work on responsive pagination?

Reply:
Possible alternatives:

- Using CSS parent slecto `:has` (not supported by Firefox: https://caniuse.com/?search=%3Ahas)
- Using Js to alter the parent according to the content of the link? (🤢)

Reply:
As said in private, it doesn't interfere with the responsive pagination.

Concerning the solution, we won't have much more solution than the one proposed here. 

Can you provide a solution using `.has-label` or equivalent to better understand the impact for the users and which one we should use ?

Reply:
Link to the requested prototype: https://codepen.io/MewenLeHo/pen/LYXWqrJ?editors=1100

First prototype (see Netlify preview):
Need to add a class to the `ul` and add this new class to style file.

Second prototype (see Codepen):
Need to add a class to the each `a` and make some changes to the actual style.

Please note that I used `.has-label` in Codepen but we need to find a new name for this class since `.has-label` was used in V4 for another purpose.

Reply:
Thanks for the protoype but it' not what I had in mind.

I had in mind a `.has-label` on the `ul` as well but with the same purpose as in v4 so the class should be added when you want to display the `Previous` ans `Next` labels. I don't know if it's feasible.

An other solution could be to remove our actual behavior with the `:empty` rule. and replace the simple text by `<span class="me-3">Previous</span>` but it could introduce a small shift.

Reply:
Viewing this a month after, I'm wondering if there isn't any way to modify `_pagination.scss` in order to have no change for end-users. I feel it's feasible and I'm wondering if it's worth a try. Any thoughts on this ? Have you tried anything in this way ?

Reply:
Tried it this way https://codepen.io/louismaximepiton/pen/zYMXqzp, do you think it's worth a try ?

Reply:
@louismaximepiton: I made the change according to your prototype. Seems interresting because there is no need to change the markup (except the small change to add an accessible name for the icons _via_ the `visually-hidden` span around each icon).

I am currently looking for any potential side effect.